frontend/src/components/roadmap/sections/RoadmapTimeline.tsx

```diff
@@ -0,0 +1,47 @@
+import { useMessages, useTranslations } from "next-intl";
+
+import RoadmapPageSection from "src/components/roadmap/RoadmapPageSection";
+
+export default function RoadmapTimeline() {
+  const t = useTranslations("Roadmap.sections.timeline");
+  const messages = useMessages() as unknown as IntlMessages;
+  const { contentItems = {} } = messages.Roadmap.sections.timeline;
```

Comment:
Not sure what I'm doing wrong here. But it's causing tests to fail. Getting this error: 

> TypeError: Cannot read properties of undefined (reading 'contentItems')

@doug-s-nava @myduong-navapbc any guidance you could provide to help get this over the finish line?

Reply:
you need to add a "timeline" item in the mocked useMessages response here:

https://github.com/HHS/simpler-grants-gov/blob/238155ff7d1dceb1dc6b5e2b7a2bd32a251fc7d9/frontend/src/utils/testing/intlMocks.ts#L27

Reply:
That was it. Thanks. 😄